VideoWeb/VideoWeb.AcceptanceTests/Steps/LoginSteps.cs

```diff
@@ -64,11 +78,12 @@ public void ThenTheSignOutLinkIsDisplayed()
 
         private bool IsAnEjudUser()
         {
-            bool isEjud = _c.VideoWebConfig.ValidEjudDIdDomains.Any(_c.CurrentUser.ContactEmail.Contains);
-            NUnit.Framework.TestContext.WriteLine($"Check for Ejud user: {_c.CurrentUser.DisplayName} is {_c.CurrentUser.GetType()} ID: {_c.CurrentUser.Id} User name: {_c.CurrentUser.Username} user type: {_c.CurrentUser.UserType} Email: {_c.CurrentUser.ContactEmail} and ejud is {isEjud}");
-            return _c.VideoWebConfig.UsingEjud &&
-                    isEjud &&
-                    (_c.CurrentUser.UserType == UserType.Judge ||
+            var isEjud = _c.VideoWebConfig.ValidEjudDIdDomains.Any(_c.CurrentUser.ContactEmail.Contains);
+            NUnit.Framework.TestContext.WriteLine($"Check for Ejud user: {_c.CurrentUser.DisplayName} is {_c.CurrentUser.GetType()} ID: {_c.CurrentUser.Id} User name: {_c.CurrentUser.Username} user type: {_c.CurrentUser.UserType} Email: {_c.CurrentUser.ContactEmail} and ejud is {isEjudEnabled}");
+            return _c.VideoWebConfig.UsingEjud 
+                && isEjud 
+                && isEjudEnabled 
+                && (_c.CurrentUser.UserType == UserType.Judge ||
```

Comment:
Consider extracting the below to a method maybe something like IsEjudgeUser
"_c.CurrentUser.UserType == UserType.Judge ||
                    _c.CurrentUser.UserType == UserType.PanelMember ||
                    _c.CurrentUser.UserType == UserType.Winger"